packages/button-react/src/Button.tsx

```diff
@@ -36,9 +42,22 @@ const makeButtonComponent = (buttonType: ValidButtons) => {
             };
 
             return (
-                <button className={componentClassName} onClick={onClick} onTouchStart={handleTouch} {...rest} ref={ref}>
-                    {children}
-                </button>
+                <ContentToggle
+                    className={classNames("jkl-button__wrapper", { "jkl-button__wrapper--compact": forceCompact })}
```

Comment:
Blir litt feil at `.jkl-button__wrapper` ligger _utenfor_ `.jkl-button`. Hvis man ikke vil gjøre breaking changes i klassenavnene for knappen er det bedre å gjøre den om til en blokk (f.eks. `.jkl.button-wrapper`).

Reply:
good point